internal/models/model_constants.go

```diff
@@ -36,6 +36,22 @@ func ProviderTypeFromString(str string) ProviderType {
 	}
 }
 
+func (pt ProviderType) SourcesProviderName() string {
+	switch pt {
+	case ProviderTypeAWS:
+		return "amazon"
+	case ProviderTypeGCP:
+		return "google"
+	case ProviderTypeAzure:
+		return "azure"
+	case ProviderTypeNoop:
+	case ProviderTypeUnknown:
+	default:
+		return ""
+	}
+	return ""
+}
```

Comment:
Heh the last three cases are not needed but I guess linter was pesky. I hate this, I am all for disabling that for the good. :-D